tests/unittests/core/test_pickleddb.py

```diff
@@ -263,6 +263,26 @@ def test_read_and_write_no_match(self, orion_db):
 
         assert loaded_config is None
 
+    def test_logging_when_getting_file_lock(self, caplog, orion_db):
+        """When logging.level is ERROR, there should be no logging."""
+        caplog.set_level(logging.INFO)
+        caplog.clear()
+        """Any operation will trigger the lock."""
```

Comment:
This should be a comment instead of a string.

Reply:
done :)